python/hail/tests/test_api.py

```diff
@@ -873,6 +873,20 @@ def test_filter_entries(self):
         entries = ds.entries()
         self.assertTrue(entries.all((entries.col_idx * entries.row_idx) % 4 == 0))
 
+    def test_to_table_on_various_fields(self):
```

Comment:
are you going to test that `.take()1` actually does the thing that you want it to do, or is that done elsewhere?

Reply:
ah, can do that for sure.